### PR TITLE
Covid 19 Plan Definition Examples in Storybook

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/react/src/stories/PlanDefinitionBuilder.stories.tsx [hunk 1]
```diff
@@ -21,3 +22,14 @@ export const Basic = (): JSX.Element => (
     />
   </Document>
 );
+
+export const Covid19Eval = (): JSX.Element => {
+  return (
+    <PlanDefinitionBuilder
+      value={Covid19CarePlanDefinition}
```
Comment (reshmakh) [inline packages/react/src/stories/PlanDefinitionBuilder.stories.tsx:29]: Looking at the storybook, looks like the PD.actions do not have timings.  Possible to add them?

**Comment by reshmakh** [comment]
Is there a way to represent that this item is done by a physician? https://medplum-storybook-8gxlou26o-medplum.vercel.app/?path=/docs/medplum-resourcetable--covid-19-review-report-activity

**Comment by reshmakh** [comment]
Show code doesn't work for this https://medplum-storybook-8gxlou26o-medplum.vercel.app/?path=/docs/medplum-resourcetable--covid-19-review-report-activity

**Comment by reshmakh** [comment]
Some additional nits on the copy for https://medplum-storybook-8gxlou26o-medplum.vercel.app/?path=/docs/medplum-plandefinitionbuilder--covid-19-eval

Could be made a bit more descriptive and in line with protocols I have seen

Title: COVID-19 Evaluation Pre-Admission to Inpatient Oncology Department
Patient Follow Up: Patient admission appointment with specialist

**Comment by rahul1** [comment]
@reshmakh I'm seeing the Show Code here. Are you seeing something different? 
image

**Comment by rahul1** [comment]
> Is there a way to represent that this item is done by a physician? https://medplum-storybook-8gxlou26o-medplum.vercel.app/?path=/docs/medplum-resourcetable--covid-19-review-report-activity

There's not a great way, but I did add a `participant` property to the action to indicate this

**Review by codyebberson — CHANGES_REQUESTED** [review]
Can we move `covid19.ts` into `packages/react/src/stories/`?  You can add the resources to the mock repository in the Storybook story.  In my opinion, we don't need to preload them into the mock repo (we should probably have a longer converasation about what should be included there).

Stories are ignored by Sonar, so that also will take care of the security hotspots and duplication warnings.

Alternatively, you can sign into Sonar and mark them as "safe" because we're not using the URL's for requests.

**Review by codyebberson — APPROVED** [review]

**Pull request merged**